data/vocab/scholar.ttl

```diff
@@ -437,6 +437,24 @@ scholar:discipline a rdf:Property ;
     rdfs:range bf:Concept ;
     rdfs:domain bf:Person .
 
+scholar:worksCount a rdf:Property ;
+    rdfs:label "works count"@en ;
+    rdfs:comment "The number of works associated with a person."@en ;
+    rdfs:range rdfs:Literal ;
+    rdfs:domain bf:Person .
+
+scholar:countsByYear a rdf:Property ;
+    rdfs:label "counts by year"@en ;
+    rdfs:comment "The number of works associated with a person organized by year."@en ;
+    rdfs:subClassOf bf:List .
+    rdfs:domain bf:Person .
```

Comment:
Cannot be a subclass of a class. Need to be changed to literal. Should be able pass in several instances doesn't have to be defined as a list